templates/redis/statefulset.yaml

```diff
@@ -33,6 +33,18 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+{{- if eq .Values.redis.type "internal" }}                                      
+      initContainers:                                                           
+      - name: "change-permission-of-directory"                                  
+        image: {{ .Values.redis.internal.initContainerImage.repository }}:{{ .Values.redis.internal.initContainerImage.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}                          
+        command: ["/bin/sh"]                                                    
+        args: ["-c", "chown -R 999:999 /var/lib/redis"]                         
```

Comment:
You can also use `redis:redis` instead of `999:999`, if you reuse the `harbor-redis` image.